core/src/main/java/info/nightscout/androidaps/plugins/pump/common/utils/ProfileUtil.kt

```diff
@@ -0,0 +1,81 @@
+package info.nightscout.androidaps.plugins.pump.common.utils
```

Comment:
to pump common?

Reply:
Yes this in several other classes are for pump commons, but I wanted to do that later... I will move the ones that can be moved without troubles, and we can do others later...